### DISCUSSION TITLE
Dialog close button doesn't close on "enter"

### DISCUSSION THREAD

**Discussion by Just-Dominic** [primefaces/community#935] [PrimeFaces] [answered]
Hi,

I want to close my dialogs by using the enter key when the button is in focus [by using tab]. It doesn't work. Did I found a bug or am I doing something wrong?

Please let me know,

Dominic

**Comment by melloware** [comment, marked as answer]
What version of PrimeFaces are you using?

It works on the Showcase: https://www.primefaces.org/showcase/ui/overlay/dialog.xhtml?jfwid=e5a55

**Comment by Just-Dominic** [comment]
Ahh I see. We are currently on V12
